draft-ietf-core-groupcomm-bis.md

```diff
@@ -541,7 +561,7 @@ As part of a message exchange between the client and any of the servers in the C
 
 ## Caching ## {#sec-caching}
 
-CoAP endpoints that are members of a CoAP group MAY cache responses to a group request as defined in {{Section 5.6 of RFC7252}}. In particular, these same rules apply to determine the set of request options used as "Cache-Key".
+CoAP endpoints that are members of a CoAP group MAY cache responses to a group request as defined in {{Section 5.6 of RFC7252}}. The set of request options used as "Cache-Key" is also as defined in {{Section 5.6 of RFC7252}}. In particular, these same rules apply to determine the set of request options used as "Cache-Key".
```

Comment:
This doesn't read right now. The last sentence "In particular, ... Cache-Key." exactly repeats the meaning of the sentence before it.   It says effectively:

 "Options used as X are defined in Y.  In particular, Y defines the options used as X."

Also not clear what "these same rules" really refers to, there are no prior rules mentioned.

Reply:
The last sentence "In particular ..." was wrongly forgotten there. It was initially intended to replace its previous version in the old text, but then the idea became to instead replace it altogether with the second sentence "The set of request options ...". Fixed in https://github.com/core-wg/groupcomm-bis/commit/3e16093ce63eb93b11e9833f1cc0bb30fc7383bc